sdks/java/extensions/avro/build.gradle

```diff
@@ -35,7 +41,12 @@ def avroVersions = [
     '1111': "1.11.1",
 ]
 
-avroVersions.each { k, v -> configurations.create("avroVersion$k") }
+avroVersions.each { k, v ->
```

Comment:
Could you elaborate the changes for Avro multiversion gradle configurations/tasks here? Why they are needed?

Reply:
Related to this comment: https://github.com/apache/beam/pull/26320#discussion_r1214111539. Will put more detailed comments.